src/PerfView/app.config

```diff
@@ -7,4 +7,7 @@
   <startup>
     <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.5"/>
   </startup>
+  <runtime>
+    <AppContextSwitchOverrides value="Switch.System.IO.UseLegacyPathHandling=false;Switch.System.IO.BlockLongPaths=false" />
```

Comment:
❔ Any reason to not just update to .NET 4.7.2?

Reply:
It depends on which OS you can get data. See https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/versions-and-dependencies
.NET 4.5 supports ✔️ 2012   | ✔️ 8

If you go to 4.7.2 you can only collect data on 

Windows versions | ✔️ 10 October 2018 Update (Version 1809)✔️ 10 April 2018 Update (Version 1803)➕ 10 Fall Creators Update (Version 1709)➕ 10 Creators Update (Version 1703)➕ 10 Anniversary Update (Version 1607)➕ 8.1➕7

✔️ Windows Server 2019✔️ Windows Server, version 1809✔️ Windows Server, version 1803➕ Windows Server, version 1709➕ 2016➕ 2012 R2➕ 2012➕ 2008 R2 SP1

Is this really what you want?

Reply:
Right, my preference would be to keep support for .NET 4.5.